porter/v1/datastore.proto

```diff
@@ -2,6 +2,57 @@ syntax = "proto3";
 
 package porter.v1;
 
+import "porter/v1/cluster.proto";
+
+// PorterDatastore is the specification for a Porter-managed datastore
+message PorterDatastore {
```

Comment:
`ManagedDatastore` - avoid using the company name where possible because rebranding becomes a PITA should it ever happen